test/pretty_printer_test.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+require_relative 'lib'
+
+describe 'PrettyPrinter tests' do
+  it 'must work with an empty list' do
+    list = [Oppen::Token::EOF.new]
+    assert_equal(
+      Oppen.pretty_print_tokens(list),
+      '',
```

Comment:
You put actual, expected but you need expected, actual.

Read the rubucop style guide on how to write tests: https://github.com/rubocop/minitest-style-guide

And try to stick with the `_` notation, so for instance, this test, I would write as:

```ruby
_(Oppen.pretty_print_tokens(list)).must_equal ''
```